### PR TITLE
test(card): add default e2e storybook tests

### PR DESCRIPTION
**Pull request by ariellalgilmore**
### Related Ticket(s)

#7449

### Description

Added Card default tests. Checking to make sure all the text renders properly and clickable, all the different storybook knobs (tags, image, outline, inverse) work properly, and checks the different theming changes the card.

### Changelog

**New**

- card e2e tests
- testing for eyebrow, heading, copy, footer text
- entire card is clickable
- tags appear 
- image appear
- outline classes applied with correct css
- inverse classes applied with correct css

**Changed**

- removed card-static test files to condense to one card e2e test

### CODE REVIEW

File: packages/react/tests/e2e-storybook/cypress/integration/Card/Card.e2e.js [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+          expect(location).to.equal(initialLocation);
+        });
+    });
+  },
+  checkImageRenders: () => {
+    cy.get(_selectors.image).should('have.length', 1);
+    cy.screenshot();
+    cy.percySnapshot(Cypress.currentTest.titlePath.join(' | '), {
+      widths: [1280],
+    });
```
Comment (jeffchew) [inline packages/react/tests/e2e-storybook/cypress/integration/Card/Card.e2e.js:137]: Can replace this with the new method:

```
cy.takeSnapshots()
```

File: packages/react/tests/e2e-storybook/cypress/integration/Card/Card.e2e.js [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+    cy.get(_selectorBase).should('have.class', `${prefix}--card--light`);
+    // converted HEX var(--cds-ui-02, #ffffff) to RGB
+    cy.get(_selectorBase)
+      .should('have.css', 'background-color')
+      .and('equal', 'rgb(255, 255, 255)');
+
+    cy.screenshot();
+    cy.percySnapshot(Cypress.currentTest.titlePath.join(' | '), {
+      widths: [1280],
+    });
```
Comment (jeffchew) [inline packages/react/tests/e2e-storybook/cypress/integration/Card/Card.e2e.js:155]: Replace with `cy.takeSnapshots()`

File: packages/react/tests/e2e-storybook/cypress/integration/Card/Card.e2e.js [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+    cy.get(_selectorBase).should('have.class', `${prefix}--card--inverse`);
+    // converted HEX var(--cds-inverse-02, #393939) to RGB
+    cy.get(_selectorBase)
+      .should('have.css', 'background-color')
+      .and('equal', 'rgb(57, 57, 57)');
+
+    cy.screenshot();
+    cy.percySnapshot(Cypress.currentTest.titlePath.join(' | '), {
+      widths: [1280],
+    });
```
Comment (jeffchew) [inline packages/react/tests/e2e-storybook/cypress/integration/Card/Card.e2e.js:167]: Replace with `cy.takeSnapshots()`

File: packages/web-components/tests/e2e-storybook/cypress/integration/card/card.e2e.js [hunk 4]
```diff
@@ -0,0 +1,280 @@
...
+    });
+  },
+  checkImageRenders: () => {
+    cy.get(_selectors.image).should('have.length', 1);
+    cy.screenshot();
+    // Take a snapshot for visual diffing
+    // TODO: click states currently not working in percy for web components
+    // cy.percySnapshot(Cypress.currentTest.titlePath.join(' | '), {
+    //  widths: [1280],
+    // });
```
Comment (jeffchew) [inline packages/web-components/tests/e2e-storybook/cypress/integration/card/card.e2e.js:146]: Replace with `cy.takeSnapshots()`

File: packages/web-components/tests/e2e-storybook/cypress/integration/card/card.e2e.js [hunk 5]
```diff
@@ -0,0 +1,280 @@
...
+  },
+  checkTagGroupRenders: () => {
+    cy.get(_selectors.tagGroup).should('have.length', 1);
+    cy.screenshot();
+    // Take a snapshot for visual diffing
+    // TODO: click states currently not working in percy for web components
+    // cy.percySnapshot(Cypress.currentTest.titlePath.join(' | '), {
+    //  widths: [1280],
+    // });
```
Comment (jeffchew) [inline packages/web-components/tests/e2e-storybook/cypress/integration/card/card.e2e.js:155]: Replace with `cy.takeSnapshots()`

**Review by IgnacioBecerra — APPROVED** [review]
It's nice seeing all these Card tests in a single file, thanks @ariellalgilmore! Once this gets merged, I can add mine from #7775 to keep it consistent 👍

File: packages/web-components/tests/e2e-storybook/cypress/integration/card/card.e2e.js [hunk 6]
```diff
@@ -0,0 +1,270 @@
...
+      .find(`.${prefix}--card__wrapper`)
+      .should('have.css', 'background-color')
+      .and('equal', 'rgb(255, 255, 255)');
+
+    cy.screenshot();
+    // Take a snapshot for visual diffing
+    // TODO: click states currently not working in percy for web components
+    // cy.percySnapshot(Cypress.currentTest.titlePath.join(' | '), {
+    //  widths: [1280],
+    // });
```
Comment (IgnacioBecerra) [inline packages/web-components/tests/e2e-storybook/cypress/integration/card/card.e2e.js:167]:
```suggestion
    cy.takeSnapshots();
```

I think you just missed these?

File: packages/web-components/tests/e2e-storybook/cypress/integration/card/card.e2e.js [hunk 7]
```diff
@@ -0,0 +1,270 @@
...
+    cy.get(_selectorBase)
+      .should('have.css', 'background-color')
+      .and('equal', 'rgb(57, 57, 57)');
+
+    cy.screenshot();
+    // Take a snapshot for visual diffing
+    // TODO: click states currently not working in percy for web components
+    // cy.percySnapshot(Cypress.currentTest.titlePath.join(' | '), {
+    //  widths: [1280],
+    // });
```
Comment (IgnacioBecerra) [inline packages/web-components/tests/e2e-storybook/cypress/integration/card/card.e2e.js:181]:
```suggestion
    cy.takeSnapshots();
```

**Pull request merged**